src/frontend/tests/sum.test.js

```diff
@@ -0,0 +1,5 @@
+import sum from '../components/sum';
```

Comment:
I understanding that this file was created in order to setup jest but it is not needed after the setup has been confirmed to work.